### PR TITLE
adds support for shoes with extra height 

### PR DESCRIPTION
**Pull request by Militaires**
done

### CODE REVIEW

**Comment by anconfuzedrock** [comment]
>slip guy
>steal shoes
>put on high heels
gg no re

**Comment by WJohn** [comment]
So these shift your whole sprite around?

That's... an interesting implication.

**Comment by ExcessiveUseOfCobblestone** [comment]
Running in Heels makes you lose both of your legs lol

**Comment by GunHog** [comment]
Put these on cuffed people so they cannot run away!

File: code/modules/clothing/clothing.dm [hunk 1]
```diff
@@ -374,6 +376,24 @@ BLIND     // can't see anything
...
+	if(offset && slot_flags & slotdefine2slotbit(slot))
+		user.pixel_y += offset
+		worn_y_dimension -= (offset * 2)
+		user.update_inv_shoes()
+		dropped_twice = TRUE
+
+/obj/item/clothing/shoes/proc/restore_offsets(mob/user)
+	dropped_twice = FALSE
+	user.pixel_y -= offset
+	worn_y_dimension = 32
```
Comment (coiax) [inline code/modules/clothing/clothing.dm]: I just have a feeling that at some point something's going to go wrong with this. And it'll be hilarious.

**Comment by Earthykiller127** [comment]
add rshoe in, automatically kicks every admins online and flood distro with plasma when worn

File: code/modules/clothing/clothing.dm (hunk 1 above)
Comment (Militaires) [inline code/modules/clothing/clothing.dm]: it's 32 relative to the human there's no process in the game that changes this other than this so

**Comment by calzilla1** [comment]
FUCK OF EARTLY

**Comment by chanoc1** [comment]
i request stilts to be added as a proof of concept

File: code/modules/clothing/clothing.dm (hunk 1 above)
Comment (RemieRichards) [inline code/modules/clothing/clothing.dm]: not 32, ```world.icon_size```

**Comment by optimumtact** [comment]
Unecessary code for a feature that has no impact imo :-1:

**Comment by Militaires** [comment]
@optimumtact might as well remove all of our particle effects and shit 

you're denying some serious possibilities.

giant stilts and tower shoes that make you impossible to shoot, add-water-to-expand clown shoes, shoes filled with monkey cubes

**Comment by Iamgoofball** [comment]
Am I going to get banned again for a griefer posting on a PR of his own free will? I didn't send him here this time but who knows

**Comment by KorPhaeron** [comment]
citrus said no

**Comment by Iamgoofball** [comment]
I'm making stilts and reopening this

**Comment by optimumtact** [comment]
I'll reopen a hole in your head mannnnnnn

**Comment by chanoc1** [comment]
rip

**Comment by Iamgoofball** [comment]
oranges imagine nuke ops on stilts

**Comment by chanoc1** [comment]
automatic stilt boots - if a projectile is coming towards you, the stilt boots activate and the just go between your legs. however, it takes a few seconds to deactivate and your movespeed is lowered during this time period

someone get on that

File: code/modules/clothing/clothing.dm [hunk 2]
```diff
@@ -359,6 +360,7 @@ BLIND     // can't see anything
 	slowdown = SHOES_SLOWDOWN
 	var/blood_state = BLOOD_STATE_NOT_BLOODY
 	var/list/bloody_shoes = list(BLOOD_STATE_HUMAN = 0,BLOOD_STATE_XENO = 0, BLOOD_STATE_OIL = 0, BLOOD_STATE_NOT_BLOODY = 0)
+	var/dropped_twice = FALSE
```
Comment (optimumtact) [inline code/modules/clothing/clothing.dm]: I don't know how I feel about this, what's causing double drops?

Comment (Militaires) [inline code/modules/clothing/clothing.dm]: hopefully it makes more sense now

Comment (Militaires) [inline code/modules/clothing/clothing.dm]: @optimumtact it's to avoid dropping the shoe from your hand and it still shifts ur pixel down

**Comment by Militaires** [comment]
file fuckery again let me fix

**Comment by Militaires** [comment]
what the fuck

**Comment by PKPenguin321** [comment]
When you commit after fixing conflicts you have to leave all the changed files ticked or this happens

**Comment by Militaires** [comment]
@PKPenguin321 kek

**Comment by Militaires** [comment]
now this must go through 24 hour rule again even though it sat for 2 weeks fuck this gay earth

**Comment by Okand37** [comment]
Darn, now how am I supposed to get some stylin' pumps?

**Comment by Militaires** [comment]
@Okand37 dont worry ill make a new pr 

basilman always delivers no matter what

**Comment by Okand37** [comment]
Excellent, fashion lives on!

**Pull request closed without merging**